tests/passes/loop_local_memory_reduction_test.py

```diff
@@ -507,12 +594,13 @@ def tester(b: dace.float64[16, 16], c: dace.float64[16]):
             for jj in range(4):
                 a[ii, jj] = ii + jj
 
+        # In theory, in this case, LLMR could be applied
```

Comment:
I suggest adding a TODO(later) to track supporting this, or explaining why it is infeasible to support